#!/usr/bin/env python
PACKAGE = "thrustController"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("altitude_feedforward", double_t, 0, "Altitude controller feedforward", 1.7, 0, 50)
gen.add("altitude_kp", double_t, 0, "Altitude controller kp gain", 0.04, 0, 50)
gen.add("altitude_ki", double_t, 0, "Altitude controller Ki gain", 0.01, 0, 50)
gen.add("altitude_kd", double_t, 0, "Altitude controller Kd gain", 0.00, 0, 50)

exit(gen.generate(PACKAGE, "thrustControllerNode", "thrustController"))
